src/views/MobileEditorHome.vue

```diff
@@ -420,13 +422,38 @@ const duplicateAudioItem = async () => {
 const shouldShowPanes = computed<boolean>(
   () => store.getters.SHOULD_SHOW_PANES
 );
+const shouldShowSidePanes = computed<boolean>(
+  () => shouldShowPanes.value && $q.screen.gt.xs
```

Comment:
なんとなく把握しときたいのですが、xsって何pxくらいでしょう。600pxくらい？
見慣れないのでコメントもあるとちょっとだけ親切かも。
```suggestion
  () => shouldShowPanes.value && $q.screen.gt.xs // 横幅がある程度大きいかどうか
```

Reply:
https://quasar.dev/style/visibility#window-width-related
600pxですね。